gossipd/gossip.c

```diff
@@ -1848,7 +1845,8 @@ static struct io_plan *conn_init(struct io_conn *conn, struct reaching *reach)
 		ai.ai_addrlen = sizeof(sin6);
 		ai.ai_addr = (struct sockaddr *)&sin6;
 		//FIXME: SAIBATO for now use allways proxy if set because we dont want to leak our ip's when using tor
```

Comment:
This seems to disagree with `doc/TOR.md`.  Can we support a mode with public IP while still able to connect to an onion LN node or not?  If not, we should remove that from `doc/TOR.md` or add support for that mode.

Tor is generally slower, so a c-lightning user might want to connect on clearnet as much as possible if a clearnet address to the node is available, and only use Tor for hidden service nodes.